frontend/ui/widget/buttondialog.lua

```diff
@@ -91,8 +92,14 @@ function ButtonDialog:init()
     end
     if self.dismissable then
         if Device:hasKeys() then
-            local close_keys = Device:hasFewKeys() and { "Back", "Left" } or Device.input.group.Back
-            self.key_events.Close = { { close_keys } }
+            local back_group = util.tableDeepCopy(Device.input.group.Back)
+            if Device:hasFewKeys() then
+                table.insert(back_group, "Left")
```

Comment:
Is it the usual behavior?
If so, can it be done when building `Device.input.group.Back` and used everywhere?

Reply:
Some widgets have a close/cancel button you can navigate to and press enter. As such my conception of it was only to add it where necessary because you would get stuck otherwise.

But indeed if it's being added here to ButtonDialog as a matter of course that principle goes out the window, since then it already is effectively everywhere.

That might be better because it ensures `Back` is available. Keep in mind hasFewKeys is something of a thought experiment. But in that case code-wise it'd make more sense to add `Left` to the `Back` group at the beginning.

There is one minor negative aspect to that, at least right now: it doesn't rejig itself when (dis)connecting a keyboard. But I suppose the devices and/or us don't support BT keyboards on hasFewKeys anyway.